src/storage/single_file_block_manager.cpp

```diff
@@ -220,7 +220,7 @@ void SingleFileBlockManager::CreateNewDatabase() {
 	max_block = 0;
 }
 
-void SingleFileBlockManager::LoadExistingDatabase() {
+void SingleFileBlockManager::LoadExistingDatabase(const idx_t block_alloc_size) {
```

Comment:
Can we directly read the block alloc size from the `BlockManager` instead of passing it to these functions?